deployment/local_deployment.md

```diff
@@ -0,0 +1,88 @@
+
+## Database Setup
+
+Most of the OCS applications depend on a postgreSQL database. The Science Archive additionally requires the postGIS plugin to be installed in its postgreSQL database. It is possible to use the same postgreSQL instance across all the applications for a simpler setup, they will just each need their own database created. Your options for running postgreSQL are:
+
+* Use an existing postgreSQL instance on your local network
+* Install postgreSQL locally on your machine
+* Run a postgreSQL docker container with postGIS installed
+
+To start a postgreSQL instance with postGIS on a machine with docker forwarding its port 5432 for local usage, you can use:
+
+```bash
+docker run --name ocs-postgis -p 0.0.0.0:5432:5432 -e POSTGRES_PASSWORD=postgres -d postgis/postgis
+```
+
+{% include notification.html message="By default, postgreSQL within a docker container will store its data in the docker container's filesystem. If you want to persist the database, should should either create a docker volume or volume mount the postgreSQL data directory in the container to a folder on your local filesystem when you start the container. Check out the [docker docs](https://docs.docker.com/storage/volumes/) for more information." %}
+
+After you have a postgreSQL instance running that you can connect to with a username and password, create a database for each OCS application you will use. These instructions assume the username and password are both set to the value `postgres`:
+
+```bash
+createdb --host 127.0.0.1 -p5432 -Upostgres -W my_db_name
+```
+
+{% include notification.html message="When running in production, it is recommended that database backups are created periodically. How frequently you create backups will depend on your needs. For example, creating one backup every day, and saving the last seven backups that were created would ensure that you would lose at most one day's worth of data, and that you could revert to any snapshot of the database taken within the last seven days. For information on how to create database backups, you can refer to the postgreSQL documentation [here](https://www.postgresql.org/docs/9.1/backup.html)."
+status="is-info"
+icon="fas fa-exclamation-triangle" %}
+
+## Docker-Compose Templates
+
+The preferred way to run each OCS application is using Docker. This way, you do not need to worry about installing the proper dependencies, or debugging strange interactions on your specific system. Template docker-compose.yaml files are provided within each OCS application repo to help getting setup to run the application in production. The docker-compose files should be setup in the repo to work and connect with one another considering a purely local deploy of all applications. If you are installing the OCS applications on different networked servers, you will need to update their environment variables to point to the correct location for each of their connections.
```

Comment:
Delete this paragraph (the one on line 38). The same information is covered in the subsequent paragraphs.